### ISSUE TITLE
Latest Wasm Bits?

### ISSUE THREAD

**Issue by TheXenocide** [stefan-schweiger/bakkBenchmark#1]
Any value in rerunning the DotNet.Wasm tests now that Blazor is officially released? I'd also be interested in knowing how that performance compares against Blazor server-side running on a local server. I'm sure AOT will make a bigger difference when they get around to solving that, but curious how much has changed in the last year.

**Comment by stefan-schweiger** [comment]
Hi, I think there would be value in rerunning the benchmark with a current build. But even more interesting would be to rerun them with the current basic AOT support (for example with https://github.com/unoplatform/Uno.Wasm.Bootstrap#aot-mode).

I tried to get the AOT running a few weeks ago, but since I don't really have time currently and it didn't work right away I didn't go through with it.

As for SSR I don't see much value in rerunning it because the performance for this benchmark will most likely be the same as for the normal .NET benchmark (because network and framework overhead will be only milliseconds).

**Comment by TheXenocide** [comment]
Fair enough on the SSR stuff. I actually stumbled on these benchmarks while chasing down information about threading support in WASM (and, in turn, Mono's threading enabled WASM). I didn't dig too deeply into these benchmarks, but I'm starting to wonder what the performance differences will be like between their single thread task scheduler and the threading-enabled runtime, though I haven't seen much trying to compare them yet. Other WASM-based threading tests seem to show improvements when written specifically for the platform, but the isolation of web workers and the nature of the messaging systems used (between cloning and SharedArrayBuffer) seems to me like depending on how Mono implements this, might have some deep non-trivial performance differences. Anyway, just kinda checking in; I have a threading issue to run down for now, but I'm interested in what you have here. Thanks for taking the time to respond!
